core/java/iesi-rest-without-microservices/src/main/resources/application-iesi-metadata.yml

```diff
@@ -1583,7 +1583,7 @@ iesi:
             description: String type parameter value
             order: 5
             type: CLOB
-            length: 4000
+            length: 6000
```

Comment:
Reflect these changes in test directory of iesi-core and iesi-rest as well as root directory of IESI i.e. iesi/core/conf

Note : Every YML file have 5 copies and so make changes in all the copies